packages/manager/apps/ips/src/pages/actions/deleteVirtualMac/deleteVirtualMac.page.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useMemo } from 'react';
+import { useNotifications } from '@ovh-ux/manager-react-components';
+import { ODS_BUTTON_VARIANT, ODS_TEXT_PRESET } from '@ovhcloud/ods-components';
+import { NAMESPACES } from '@ovh-ux/manager-common-translations';
+import {
+  OdsButton,
+  OdsModal,
+  OdsSpinner,
+  OdsText,
+} from '@ovhcloud/ods-components/react';
+import {
+  ButtonType,
+  PageLocation,
+  PageType,
+  useOvhTracking,
+} from '@ovh-ux/manager-react-shell-client';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import {
+  useDeleteVirtualMac,
+  useGetIpdetails,
+  useGetIpVmacWithIp,
+} from '@/data/hooks/ip';
+import { fromIdToIp, ipFormatter } from '@/utils';
+
+export default function DeleteVirtualMac() {
+  const navigate = useNavigate();
+  const { id } = useParams();
+  const { ip } = ipFormatter(fromIdToIp(id));
+  const { ipDetails } = useGetIpdetails({ ip });
+  const { t } = useTranslation('listing');
```

Comment:
```suggestion
  const { t } = useTranslation(TRANSLATION_NAMESPACES.listing);
```

There is a constant to avoid typos on namespaces